CrowdStrike/ember-toucan-core
# Update docs-app to be more fleshed out and production-ready

## 🚀 Description

This PR gets the docs-app in a more publishable state by:

- Updating the `<meta>` `<description>` so that when the link is shared, a more proper description is displayed over what we have today
- Removing the feature cards for now until we figure out what we want to put here
- Adding the CrowdStrike logo now that we are under the Apache v2 license
- Adding a GitHub link to the repo
- Updating the way we specify "Optional" by putting it to the front
- Updating all UI State sections to roughly match
  - **NOTE:** Some components are trickier and have more args than others, so not an exact 1:1
  - **NOTE2:** I did not touch InputFile and Fieldset as those components are getting a few changes and I want to prevent merge conflicts at the moment
- Updating all demos to be more "real"

---

## 🔬 How to Test

- Visit https://ec41bb62.ember-toucan-core.pages.dev
- Verify the CrowdStrike logo in the top-left. Click it and verify it goes to our website.
- Verify the GitHub logo in the top-right.  Click it and verify it goes to this repo.
- Verify the main page no longer has "Coming soon" text
- Verify in the PR that "Optional." is now at the front and replaces "This is optional."
- Browse the UI state demos touched here and verify they're in good shape
  - **NOTE:** Ignore InputFile and Fieldset (see above)

---

## 📸 Images/Videos of Functionality

Images are in the PR when reviewing!
